src/test/feedback/commands/submitFeedbackListener.test.ts

```diff
@@ -0,0 +1,116 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import { FeedbackPanel, submitFeedbackListener, Window } from '../../../feedback/commands/submitFeedbackListener'
+import { TelemetryFeedback } from '../../../shared/telemetry/telemetryFeedback'
+import { TelemetryService } from '../../../shared/telemetry/telemetryService'
+import { TelemetryEvent } from '../../../shared/telemetry/telemetryEvent'
+
+class MockFeedbackPanel implements FeedbackPanel {
```

Comment:
nit: these are fakes (actually implements functionality, with a non-production impl), not mocks. Mocks are useless things that say how many times you called a method, etc.. :)

The distinction is useful to me because "fake" tells me this is an actual impl, not something using mocking library. I can meaningfully step-into a fake via the debugger, whereas with a mock I get the meaningless guts of a mocking library.

Reply:
Lol I agree, I'm usually the one to post this comment for others. I refer to the [Fowler article](https://martinfowler.com/articles/mocksArentStubs.html).

For this one, I had copied the naming from another file.

I do usually refer to these as "fakes", however I've started to realize they're more properly called "stubs". The distinction seems to be that fakes are "fully functional" implementations (The example given is an in-memory database that mimics a real database). Whereas stubs just have canned responses. 

I'm going to see if I can replace these with stubs using Sinon to remove the verbosity.